homeassistant/components/rako/bridge.py

```diff
@@ -0,0 +1,124 @@
+"""Module representing a Rako Bridge."""
+from __future__ import annotations
+
+import asyncio
+from asyncio import Task
+import logging
+
+from python_rako.bridge import Bridge
+from python_rako.helpers import convert_to_brightness, get_dg_listener
+from python_rako.model import ChannelStatusMessage, SceneStatusMessage, StatusMessage
+
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .light import RakoLight
+from .model import RakoDomainEntryData
+from .util import create_unique_id
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RakoBridge(Bridge):
+    """Represents a Rako Bridge."""
+
+    def __init__(
+        self,
+        host: str,
+        port: int,
+        name: str,
+        mac: str,
+        entry_id: str,
+        hass: HomeAssistant,
+    ) -> None:
+        """Init subclass of python_rako Bridge."""
+        super().__init__(host, port, name, mac)
+        self.entry_id = entry_id
+        self.hass = hass
+
+    @property
+    def _light_map(self) -> dict[str, RakoLight]:
+        rako_domain_entry_data: RakoDomainEntryData = self.hass.data[DOMAIN][self.mac]
+        return rako_domain_entry_data["rako_light_map"]
+
+    @property
+    def _listener_task(self) -> Task | None:
+        rako_domain_entry_data: RakoDomainEntryData = self.hass.data[DOMAIN][self.mac]
+        return rako_domain_entry_data["rako_listener_task"]
+
+    @_listener_task.setter
+    def _listener_task(self, task: Task) -> None:
+        rako_domain_entry_data: RakoDomainEntryData = self.hass.data[DOMAIN][self.mac]
+        rako_domain_entry_data["rako_listener_task"] = task
```

Comment:
The listener task seems like it could be a member variable of this class. Is there a particular motivation to set it on the data object that i'm missing or would that work?